util/envDynawo.sh

```diff
@@ -1915,6 +1922,7 @@ create_distrib() {
   zip -r -y $ZIP_FILE bin/ lib/ share/ sources/
   zip -r -g -y $ZIP_FILE ddb/*.$DYNAWO_SHARED_LIBRARY_SUFFIX ddb/*.desc.xml ddb/*.extvar
   zip -r -g -y $ZIP_FILE sbin/curvesToHtml
+  zip -r -g -y $ZIP_FILE sbin/xsl
```

Comment:
To add in create_modelica_distrib also
And the script need nry.py also ?

Reply:
Not needed in create_modelica_distrib as we zip the whole sbin folder 
I am going to add the nrt script